products/revenue_analytics/backend/views/revenue_analytics_charge_view.py

```diff
@@ -74,7 +74,7 @@ def for_events(cls, team: "Team") -> list["RevenueAnalyticsBaseView"]:
 
             query = ast.SelectQuery(
                 select=[
-                    ast.Alias(alias="id", expr=ast.Field(chain=["uuid"])),
+                    ast.Alias(alias="id", expr=ast.Call(name="toString", args=[ast.Field(chain=["uuid"])])),
```

Comment:
Unrelated bugfix, solves the problem when we query both events and revenue sources at the same time. That didnt happen often before, but it's always happening now that we're filtering differently.